authentication/metrics.go

```diff
@@ -9,7 +9,7 @@ func RegisterTenantsFailingMetric(reg prometheus.Registerer) *prometheus.Counter
 	return promauto.With(reg).NewCounterVec(prometheus.CounterOpts{
 		Namespace: "observatorium",
 		Subsystem: "api",
-		Name:      "tenants_failed_registrations",
+		Name:      "tenants_failed_registrations_total",
```

Comment:
WoW this is a dealbreaker of prom naming conventions we are supposed to be an observability team :dagger:

Reply:
😅